### PR TITLE
Update contact in Linux package info

### PR DESCRIPTION
**Pull request by rundong08**

### CODE REVIEW

File: distribution/linux/debian-changelog.tpl [hunk 1]
```diff
@@ -2,4 +2,4 @@ taqosurvey (1.0-1) UNRELEASED; urgency=low
 
   * Initial Release.
 
- -- Bob Evans <bobevans@google.com>  $DATE
+ -- Paco Maintainers <paco-maintainers@google.com>  $DATE
```
Comment (BobEvans) [inline distribution/linux/debian-changelog.tpl]: Double checking. This address has been created, right? :-)

Comment (BobEvans) [reply distribution/linux/debian-changelog.tpl]: Actually, we already have paco-users@googlegroups.com. We should use that address.

Comment (rundong08) [reply distribution/linux/debian-changelog.tpl]: Just updated the address to support@pacoapp.com, as discussed in chat.

**Comment by BobEvans** [comment]
Thanks!

**Pull request merged**